products/BellHybrid/apps/application-bell-main/windows/BellBatteryStatusWindow.cpp

```diff
@@ -15,7 +15,8 @@ namespace
 {
     constexpr auto imageType = gui::ImageTypeSpecifier::W_G;
     constexpr auto batteryEntries =
-        std::array<battery_utils::BatteryLevelEntry, 6>{{{{5, 10}, {1, 5}, "bell_status_battery_lvl0"},
+        std::array<battery_utils::BatteryLevelEntry, 7>{{{{0, 5}, {1, 1}, "bell_status_battery_lvl0"},
+                                                         {{5, 10}, {1, 5}, "bell_status_battery_lvl0"},
```

Comment:
No, because those two addresses other scenarios. First range is constantly showing 1%. It is because we need to display something if the device it shutdown because of low battery level and user connects it to usb and powers it up.